LoopBack/LBPersistedModel.h

```diff
@@ -7,12 +7,13 @@
 
 #import "LBModel.h"
 
+
+/**
+ * Blocks of this type are executed for a successful method invocation that returns
+ * an LBPersistedModel instance as a callback argument.
+ */
 @class LBPersistedModel;
 typedef void (^LBPersistedModelObjectSuccessBlock)(LBPersistedModel *model);
-typedef void (^LBPersistedModelArraySuccessBlock)(NSArray *array);
-typedef void (^LBPersistedModelVoidSuccessBlock)();
-typedef void (^LBPersistedModelBoolSuccessBlock)(BOOL boolean);
-typedef void (^LBPersistedModelNumberSuccessBlock)(NSInteger number);
```

Comment:
Following @bajtos 's suggestion, I've left the typedefs with deprecation warning since I'm informed that Ver. v1.3.2  has been recently which includes all the typedefs (which I didn't realise until yesterday).